docs/reference/query-languages/esql/esql-lookup-join.md

```diff
@@ -21,6 +21,7 @@ For example, you can use `LOOKUP JOIN` to:
 * Working with regular indices
 * Need to preserve distinct matches
 * Need to match on any field in a lookup index
+* You use document or field level security
```

Comment:
Or if we want to restrict users to specific lookup indices that they can use. (ENRICH is all or nothing.)